src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/Controllers/Domain/TournamentsController.cs

```diff
@@ -280,6 +281,24 @@ public ActionResult DeleteConfirmed(int id)
             _tournamentService.Delete(id);
             return RedirectToAction("Archived");
         }
+        
+        /// <summary>
+        /// Activate tournament
+        /// </summary>
+        /// <param name="id">Identifier of tournament to activate</param>
+        /// <returns></returns>
+        public ActionResult Activate(int id)
+        {
+            var tournament = _tournamentService.Get(id);
```

Comment:
Why do you need this call?

Reply:
I do understand that service already has a check for the case of null tournament. Should I change this method so it would use try catch?

Reply:
yes, why to call db extra time.